### PR TITLE
Add spacing between text and badges in sidebar

### LINKED ISSUE #1336: On the nav sidebar, add some spacing between the title and notification bubble

### PR DESCRIPTION
**Pull request by caramelcat**
Fixes #1336

### CODE REVIEW

**Comment by caramelcat** [comment]
Before:
before-badge


After:
after-badge

File: app/helpers/application_sidebar_helper.rb [hunk 1]
```diff
@@ -52,7 +52,10 @@ def link_to_sidebar_item(item)
       concat(item[:title])
       concat(
         content_tag(:span, class: ['unread']) do
-          badge(item[:unread]) if item[:unread] && item[:unread] > 0
+          if item[:unread] && item[:unread] > 0
+            concat(' ')
+            concat(badge(item[:unread])) if item[:unread] && item[:unread] > 0
```
Comment (allenwq) [inline app/helpers/application_sidebar_helper.rb]: This `if` is duplicated

**Comment by allenwq** [comment]
@caramelcat Actually why not use CSS to do this ? it's a margin-left to the span of unread.

**Comment by caramelcat** [comment]
@allenwq Sorry, too tired :'(

Removed the extraneous if.

The Bootstrap docs also use space, want to leave it as that? If you prefer margin, that's fine too. Just let me know.

**Comment by allenwq** [comment]
@caramelcat We can stick to current way since you already implemented.  Thanks for helping us !

**Pull request merged**